profiles/common/modules/features/nexteuropa_feature_set/nexteuropa_feature_set.install

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * @file
+ * Implements some specific logic for platform feature sets.
+ */
+
+/**
+ * Implements hook_install().
+ *
+ * Grant access to features set and admin menu to user role.
+ */
+function nexteuropa_feature_set_install() {
+  multisite_config_service('user')->createRole('cem', 4);
```

Comment:
@Fefaine we could also try to consider if it wouldn't be better to move this to the cce_basic_config thus the 'cem' role for me seems to be more a generic role rather than the one which is related only to the feature sets.